ovs/extensions/generic/sshclient.py

```diff
@@ -523,20 +523,27 @@ def file_write(self, filename, contents, mode='w'):
         :param contents: Contents to write to the file
         :param mode: Mode to write to the file, can be a, a+, w, w+
         """
+        temp_filename = '{0}~'.format(filename)
         if self.is_local is True:
-            with open(filename, mode) as the_file:
+            with open(temp_filename, mode) as the_file:
```

Comment:
what happens if a bigger temp_file_name file still exists when this operation is started?